pkg/minikube/constants/constants_linux.go

```diff
@@ -18,7 +18,14 @@ limitations under the License.
 
 package constants
 
+import (
+	"k8s.io/client-go/util/homedir"
+)
+
 var SupportedVMDrivers = [...]string{
 	"virtualbox",
 	"kvm",
 }
+
+var DefaultMountDir = homedir.HomeDir()
```

Comment:
This is the dir that will be mounted into the VM.  So this will map the users home directory into the VM.  If this were constants.DefaultMinipath onlint $HOME/.minikube would be mounted.  I will keep this as for now